### DISCUSSION TITLE
Onscreen Overlay Position

### DISCUSSION THREAD

**Discussion by cloaked2222** [SaifAqqad/AHK_MicMute#43] [Ideas]
Users can change the onscreen feedback position. Please also add an option to set the onscreen overlay position.

**Comment by SaifAqqad** [comment]
You can do that already, press <kbd>Ctrl</kbd> <kbd>Alt</kbd> <kbd>F10</kbd> and drag it to whatever position you want then press <kbd>Ctrl</kbd> <kbd>Alt</kbd> <kbd>F10</kbd> again to lock it

**Reply by cloaked2222** [reply]
Thank you! I'd suggest adding this to the UI. Under Edit Configuration there's currently a "OSD position" option. Just need a "Overlay position" option.
